<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>init_game</class>
 <widget class="QWidget" name="init_game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>701</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
    <italic>true</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>One Card</string>
  </property>
  <widget class="QLabel" name="init_title">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>28</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:28pt; font-weight:700;&quot;&gt;I N I T&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>616</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <italic>false</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> 
QPushButton{
		background-color: #6ab8f7;
        color: black;   
        border-radius: 10px;  
		border: 2px groove gray;
        border-style: outset;
}

QPushButton:hover {
    background-color: #ecf5ff;
    color: #409eff;
}

QPushButton:pressed, QPushButton:checked {
    border: 1px solid #3a8ee6;
    color: #409eff;
}</string>
   </property>
   <property name="text">
    <string>&lt; back to home</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next_btn">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>616</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <italic>false</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{
		background-color: #6ab8f7;
        color: black;   
        border-radius: 10px;  
		border: 2px groove gray;
        border-style: outset;
}

QPushButton:hover {
    background-color: #ecf5ff;
    color: #409eff;
}

QPushButton:pressed, QPushButton:checked {
    border: 1px solid #3a8ee6;
    color: #409eff;
}</string>
   </property>
   <property name="text">
    <string>next &gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="player_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;player number&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="deck_label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>140</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;deck number&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="init_card_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>290</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;init card number&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="round_label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>290</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;round number&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="auto_check_btn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>450</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>111</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>-1</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
     spacing: 8px;  
	font-family:Consolas;
	font-size:20px;
	border-radius:2px;
}

QCheckBox::indicator {  
        width: 15px; 
        height: 15px;
}</string>
   </property>
   <property name="text">
    <string>auto</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="log_check_btn">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>500</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>-1</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
     spacing: 8px;  
	font-family:Consolas;
	font-size:20px;
	border-radius:2px;
}
QCheckBox::indicator {  
        width: 15px; 
        height: 15px;
}</string>
   </property>
   <property name="text">
    <string>open logger</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="player_num_editor">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>14</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
    background:transparent;
    border:none;
    border-bottom:1px solid black;
}
 
QLineEdit:hover{
    border-bottom:1px solid rgb(193,193, 193);
}
 
QLineEdit:focus{
    border-bottom:1px solid rgb(18, 183, 245);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>player num</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="init_card_num_editor">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>300</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>14</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
    background:transparent;
    border:none;
    border-bottom:1px solid black;
}
 
QLineEdit:hover{
    border-bottom:1px solid rgb(193,193, 193);
}
 
QLineEdit:focus{
    border-bottom:1px solid rgb(18, 183, 245);
}</string>
   </property>
   <property name="placeholderText">
    <string>init num</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="deck_num_editor">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>150</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>14</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
    background:transparent;
    border:none;
    border-bottom:1px solid black;
}
 
QLineEdit:hover{
    border-bottom:1px solid rgb(193,193, 193);
}
 
QLineEdit:focus{
    border-bottom:1px solid rgb(18, 183, 245);
}</string>
   </property>
   <property name="placeholderText">
    <string>deck num</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="round_num_editor">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>300</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>14</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
    background:transparent;
    border:none;
    border-bottom:1px solid black;
}
 
QLineEdit:hover{
    border-bottom:1px solid rgb(193,193, 193);
}
 
QLineEdit:focus{
    border-bottom:1px solid rgb(18, 183, 245);
}</string>
   </property>
   <property name="placeholderText">
    <string>round num</string>
   </property>
  </widget>
  <widget class="QLabel" name="player_icon">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>41</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>res/player.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="deck_icon">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>140</y>
     <width>58</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>res/deck.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="init_card_icon">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>58</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>res/init_no.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="round_icon">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>290</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>res/round.png</pixmap>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
